net/pfSense-pkg-frr/files/usr/local/pkg/frr/inc/frr_bgp.inc

```diff
@@ -609,15 +609,16 @@ function frr_bgp_generate_neighbors() {
 			$tconf .= "\n";
 		}
 
-		if (frr_validate_intrange($neighbor['bfd_multiplier'], 2, 255)) {
-			$nconf .= "  neighbor {$neighbor['peer']} bfd {$neighbor['bfd_multiplier']}";
-			if (frr_validate_intrange($neighbor['bfd_minrx'], 50, 60000)) {
-				$nconf .= " {$neighbor['bfd_minrx']}";
-				if (frr_validate_intrange($neighbor['bfd_mintx'], 50, 60000)) {
-					$nconf .= " {$neighbor['bfd_mintx']}";
-				}
+		if (!empty($neighbor['bfdenable'])) {
+			if (!empty($neighbor['bfd_peer']) && $neighbor['bfd_peer'] <> 'default') {
```

Comment:
The second test here `$neighbor['bfd_peer'] <> 'default'` should be in its own set of parenthesis.